ConorMFinn/CO600-SeriousCyberSec
# Add Character Customization

**Description of work.**
Sherry is now available in a range of colours!
A menu has been added to the main menu to allow the player to choose the colour they desire.

**To test:**
Start the game, select a colour from the customization menu and check it has changed when the game is started. 

Code review: `HallStart.cs`, `CustomizeController.cs`, `ColourManager.cs`, and `MainMenuController.cs`

Fixes #94
